src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -252,4 +254,9 @@ public void createProjectForExercise(ProgrammingExercise programmingExercise) th
             throw new JenkinsException("Error creating folder for exercise " + programmingExercise, e);
         }
     }
+
+    @Override
+    public CheckoutDirectoriesDTO getCheckoutDirectories(ProgrammingLanguage programmingLanguage, boolean checkoutSolution) {
+        throw new UnsupportedOperationException("Method not implemented, consult the build plans in Jenkins for the checkout directories.");
```

Comment:
Wouldnt it be better to say. "Method not implemented, consult the build plans in Jenkins to receive more information on the checkout directories."

Reply:
I adjusted it to "Method not implemented, consult the build plans in GitLab for more information on the checkout directories." - I think this fixes the issue?

Reply:
yes thanks :)